components/feature/addons/src/test/java/mozilla/components/feature/addons/update/DefaultAddonUpdaterTest.kt

```diff
@@ -141,6 +141,57 @@ class DefaultAddonUpdaterTest {
         updater.updateStatusStorage.clear(context)
     }
 
+    @Test
+    fun `onUpdatePermissionRequest - should not show a notification for not valid permissions`() {
```

Comment:
nit: `should not show a notification for not unknown permissions`, or invalid permissions.